### ISSUE TITLE
Remove wierd error message "Error loading webapps" in class AzureRefreshableNode

### ISSUE THREAD

**Issue by t-rufang** [microsoft/azure-tools-for-java#2532]
```java
public abstract class AzureRefreshableNode extends RefreshableNode {
    // hiding not related codes
 
    @Override
    protected void onNodeClick(NodeActionEvent e) {
        try {
            if (AuthMethodManager.getInstance().isSignedIn()) {
                super.onNodeClick(e);
            }
        } catch (Exception ex) {
            DefaultLoader.getUIHelper().logError("Error loading webapps", ex);
        }
    }
}
```

As you can see, `Error loading webapps` error message is thrown when error loading `AzureRefreshableNode `

**Comment by t-rufang** [comment]
@Flanker32 Could you please help to take a look at this issue?

**Comment by Flanker32** [comment]
@t-rufang Thanks, will looking into this issue.

**Comment by Flanker32** [comment]
Fixed in #3134
